ci/plugins/mzcompose/hooks/pre-exit

```diff
@@ -20,23 +20,19 @@ run() {
 }
 
 run logs --no-color > services.log
-buildkite-agent artifact upload services.log
-
 # shellcheck disable=SC2024
 sudo journalctl --merge > journalctl-merge.log
-buildkite-agent artifact upload journalctl-merge.log
-
 netstat -ant > netstat-ant.log
-buildkite-agent artifact upload netstat-ant.log
-
 netstat -panelot > netstat-panelot.log
-buildkite-agent artifact upload netstat-panelot.log
-
 ps aux > ps-aux.log
-buildkite-agent artifact upload ps-aux.log
-
 docker ps -a --no-trunc > docker-ps-a.log
-buildkite-agent artifact upload docker-ps-a.log
+
+ARTIFACTS=(run.log services.log journalctl-merge.log netstat-ant.log netstat-panelot.log ps-aux.log docker-ps-a.log)
+for ARTIFACT in "${ARTIFACTS[@]}"; do
+  buildkite-agent artifact upload "$ARTIFACT"
+done
+
+bin/ci-flake-detect --name=mzcompose "${ARTIFACTS[@]}"
```

Comment:
In order to avoid the

```


fatal: python v3.9.0+ required
--
  | hint: you have v3.7.16
```

error, this needs to be run inside the `ci-builder` container using the  `bin/ci-builder`.  This is what the `run` shell function that is being used elsewhere in the file does.

Reply:
By running it inside the ci-builder, do we still have access to the logs directory?

Reply:
I believe so, yes. The logs are all placed in the `pwd` of the buildkite process, which I assume is the directory the materialized repo is checked out in. bin/ci-builder is set so that the `pwd` inside the container is that very same directory. The `ci-builder` container is started with:

```
            --volume "$(pwd):$(pwd)"
            --workdir "$(pwd)"
```